src/store/persona/index.ts

```diff
@@ -0,0 +1,6 @@
+import { useSelector } from 'react-redux';
+import { personaSelector } from './selector';
+
+export type { initialState as PersonaInitialState } from './types';
+export { default, PersonaState } from './slice';
+export const usePersona = () => useSelector(personaSelector);
```

Comment:
why not just using using the selector only? I am confused with `useSelector(personaSelector)`

Reply:
It the same as in user

```
import { useSelector } from "react-redux";
import { userSelector } from "./selector";

export type { initialState as UserInitialState } from "./types";
export { default, UserState } from "./slice";
export const useUser = () => useSelector(userSelector);
```
I think it like a mesmorize function in our case. We kinda standardize the selector file.